src/nested_ragged_tensors/ragged_numpy.py

```
@@ -763,42 +869,169 @@
+            >>> dense_dict['val']
+            array([[1. , 0.2, 0. , 3.1, 0. , 1. , 2.2],
+                   [3. , 3.3, 2. , 0. , 0. , 0. , 0. ]], dtype=float32)
+            >>> J.flatten(dim=0)
+            Traceback (most recent call last):
+                ...
+            ValueError: Only supports dim = -1 or 2 for now; got 0
+        """
+        if dim < 0:
+            target_dim = self.max_n_dims + dim
+        else:
+            target_dim = dim
```

Ruff

Use ternary operator `target_dim = self.max_n_dims + dim if dim < 0 else dim` instead of `if`-`else`-block

Replace `if`-`else`-block with `target_dim = self.max_n_dims + dim if dim < 0 else dim`

(SIM108)